jsonrs/json.go

```diff
@@ -0,0 +1,135 @@
+package jsonrs
```

Comment:
I'd simply call it `json` to be honest. It doesn't need to have any reference to rudder-server.

Reply:
Can we keep the original package name, `jsonrs`? 
I prefer to be no ambiguity between this and the std json package. 
It also makes it possible to enforce some linting rules like [these](https://github.com/rudderlabs/rudder-server/blob/master/.golangci.yml#L34-L46)